crates/bevy_render/src/mesh/mesh.rs

```diff
@@ -634,3 +700,45 @@ fn update_entity_mesh(
         render_pipelines.bindings.vertex_attribute_buffer = Some(vertex_attribute_buffer_resource);
     }
 }
+
+#[cfg(test)]
+mod test {
+    use crate::mesh::shape;
+
+    use super::*;
+
+    #[test]
+    #[allow(clippy::float_cmp)]
+    fn attribute_multi_mut_works_with_positions_and_normals() {
+        let mut mesh = Mesh::from(shape::Box::default());
+
+        let [positions, normals] =
+            mesh.attribute_multi_mut([Mesh::ATTRIBUTE_POSITION, Mesh::ATTRIBUTE_NORMAL]);
+
+        let positions = match positions.unwrap() {
+            VertexAttributeValues::Float32x3(arr) => arr,
+            _ => panic!("Positions were not [f32; 3]."),
+        };
+
+        let normals = match normals.unwrap() {
+            VertexAttributeValues::Float32x3(arr) => arr,
+            _ => panic!("Normals were not [f32; 3]."),
+        };
+
+        // Mutating positions and normals.
+        // Just setting them equal to each other for the heck of it.
+        normals[0] = [0.1, 0.2, 0.3];
+        positions[0] = normals[0];
+        assert_eq!(normals[0], positions[0])
+    }
+
+    #[test]
+    #[should_panic]
+    fn attribute_multi_mut_panics_on_duplicate_names() {
```

Comment:
I would like to see a test case where the names are not identical, but point to the same thing.

Reply:
As mentioned above, not sure I see how this is possible. Could you elaborate?